packages/actor-query-operation-leftjoin/test/ActorQueryOperationLeftJoin-test.ts

```diff
@@ -215,6 +215,8 @@ describe('ActorQueryOperationLeftJoin', () => {
           // Do nothing
         });
       });
+      // TODO: See if this should be closed on the error event
+      output.bindingsStream.destroy();
```

Comment:
This is a case where it may be better to update the core behavior to close the iterator on an error event.